sunpy/data/_sample.py

```diff
@@ -66,6 +66,9 @@ def download_sample_data(progress=True, overwrite=True):
         Show a progress bar during download
     overwrite: bool
         If exist overwrites the downloaded sample data.
+    timeout: float
+        The timeout in seconds.
+        Default timeout is `astropy.utils.data.Conf.remote_timeout`.
```

Comment:
Per discussion, change this line to say "If None, use the timeout specified in..."